MachineLearning_Engine/Query/numpy.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Python.Runtime;
+
```

Comment:
Also this file should probably sit in the `Objects` folder? This isn't a standard BHoM query method and doesn't meet compliance for it?

Reply:
We have `public static` fields in the engine sometimes. How do we handle them? Maybe I can comply to that.

E.g.
https://github.com/BHoM/BHoM_Engine/blob/30240fdfe964255232013497ea86fcba48ce65a2/Reflection_Engine/Query/AssemblyList.cs

Reply:
That's not really an object, it needs to be declared in an class.
I can put them in a file called `Modules`, which is what both `pyCompute` and `numpy` are, but that would be a non compliant name.

Reply:
The `Objects` folder in an Engine is for things which don't comply to the general guidelines but are needed. So the file structure would be `MachineLearning_Engine/Objects/pyCompute.cs` - but these wouldn't reflect into the UIs. If it does need to reflect into UIs then I'm happy with it being left as is for now and we can revisit in a wider discussion out of this PR :smile:

Reply:
Yeah, the problem is that it would be unintuitivy to have `pyCompute` in the `Query` class, but in the `Object` folder.

Reply:
Then for the purposes of keeping this PR to scope and allowing the work of this toolkit to continue I can tolerate it being there for now and we can revisit another day if you'd prefer :smile:

Reply:
I made them private and put the in the `Compute/Invoke.cs` file. I think this complies with the Reflection_Engine at least - I prefer it that way. Does that work for you?

Reply:
It does :smile: